baserun/instrumentation/instrumented_wrapper.py

```diff
@@ -87,7 +87,8 @@ async def instrumented_function(*args, **kwargs):
         finally:
             if auto_end_span:
                 span.end()
-                parent_span.end()
+                if parent_span.is_recording() and not Baserun.current_test_suite:
```

Comment:
I assume this should check is_recording like we do below?

Reply:
Yeah good catch